src/main/python/datastore/house_dao.py

```diff
@@ -0,0 +1,25 @@
+from pathlib import Path
+import sys
+
+import pymysql
+
+sys.path.insert(1, str(Path(__file__).parent.parent))
+from config import configuration
+
+def get_all_houses():
+    configs = configuration.fetch_database_configs(configuration.DEVELOPMENT_CONFIG_FILE_PATH)
+    connection = pymysql.connect(host=configs['host'],
+                                 user=configs['user'],
+                                 password=configs['password'],
+                                 db=configs['database'])
+
+    try:
+        cur = connection.cursor()
+        cur.execute("select * from house")
```

Comment:
Oops missed this one. Saw your Slack comment about how there's 26 million rows. So I'd rewrite this to allow dynamic field selection in the HTTP call, as well as a dynamic `LIMIT` value, both with sane defaults.

Reply:
Fixed

Reply:
(By adding limit 10 for now).  It's a quick fix on an API method that'll get updated as we go anyway.  I'm not going to put a ton of effort into what is essentially a POC.